### ISSUE TITLE
Several warnings from Tensorflow during VTAB experiments.

### ISSUE THREAD

**Issue by youngryan1993** [KMnP/vpt#29]
Thanks for the wonderful code and paper!

I am trying to run VTAB experiments. But I got the error message below. 

The code can be running on sever, but it seems they mostly use CPU and the performance is bad (I follow vtab-dmlab in your demo.md, but the performance is around 20%).

does this error message affect the performance?

Thanks for your help in advance :)

``
[01/24 22:36:29 visual_prompt]: Loading training data (final training data for vtab)..

2023-01-24 22:36:29.146201: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA

To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

2023-01-24 22:36:29.319868: I tensorflow/core/util/port.cc:104] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDN N_OPTS=0`.

2023-01-24 22:36:30.140945: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; 

2023-01-24 22:36:30.141141: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; 

2023-01-24 22:36:30.141192: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.

2023-01-24 22:36:32.136349: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; 

2023-01-24 22:36:32.136564: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudnn.so.8'; dlerror: libcudnn.so.8: cannot open shared object file: No such file or directory; 

2023-01-24 22:36:32.136631: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1934] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
``

**Comment by youngryan1993** [comment]
(+) if I run dmlab dataset with the given configuration from the author, it provides the following output.


<img width="1118" alt="Screen Shot 2023-01-25 at 7 30 07 PM" src="https://user-images.githubusercontent.com/41351363/214726096-49123225-a68e-49c6-ae88-bd0066cf8e6a.png">

[01/25 17:54:11 visual_prompt]: encountered nan loss, skip gradient updating for this batch!

**Comment by youngryan1993** [comment]
also, this generates nan loss every trial.

<img width="572" alt="Screen Shot 2023-01-25 at 10 50 53 PM" src="https://user-images.githubusercontent.com/41351363/214754974-3c0bae76-3091-4ac3-a989-dcccaa0f7326.png">

**Comment by fanlulin** [comment]
Can I run this project on windows?

**Comment by weeknan** [comment]
@youngryan1993 
Hi, I have these warnings from Tensorflow as yours. How did you solve this question?
